src/test/java/apprenticeship/GameTest.java

```diff
@@ -0,0 +1,4 @@
+package apprenticeship;
+
+public class GameTest {
+}
```

Comment:
@vincentstorme can comment on this further with his opinion, but I will give my two cents here. As your code stands right now, it is pretty tough to test the `play` method in Game. You did not do anything bad or wrong - this method is just kind of tough to test in general. The way that I have gotten around this problem in the past is by doing two things:

1. Create a public method that takes a single move. In your case, it might include `makeMove`, `renderBoard`, and check if game is in play. You can definitely test this method and then use it within the while loop.

2. Program a basic computer player that takes the first available spot on the board. Then, have two of these computer players play a game of Tic Tac Toe against each other. You can then make sure the final state of the board is correct. You don't need to do this right now! We will handle computer players in a future story.